docs/advanced/lists.md

```diff
@@ -0,0 +1,227 @@
+## Lists
```

Comment:
While migrating, we should also replace Python terms with JS terms, i.e., `list` -> `array`, `attr` -> `field`/`method`, etc.